basics/node.md

```diff
@@ -0,0 +1,26 @@
+# Interoperating with Node and NPM
+
+While Deno is pretty powerful itself, many people will want to leverage code and
+libraries that are built for [Node](https://nodejs.org/), in particular the
```

Comment:
Probably should be consistent with use of Node.  Officially (though I know Ryan argues this) it is _Node.js_ and other places we are using it.

Reply:
ok I'm happy to replace Node with Node.js in all places if you guys want to do that?

Reply:
@kitsonk

Reply:
ok these have been replaced.